runtime/doc/diagnostic.txt

```diff
@@ -464,7 +464,10 @@ get({bufnr}, {opts})                                    *vim.diagnostic.get()*
                  • namespace: (number) Limit diagnostics to the given
                    namespace.
                  • lnum: (number) Limit diagnostics to the given line number.
+                 • col: (number) Limit diagnsotics to the given column number.
                  • severity: See |diagnostic-severity|.
+                 • original: (boolean) if true the diagnostic use lsp orignial
+                   format.
```

Comment:
Not all diagnostics are LSP diagnostics, and LSP diagnostics should not be treated specially. They are one consumer of possibly many.

Reply:
oh make sense I forgot that.  so called lsp instead of original?.or something else

Reply:
when this option is true the `vim.diagnostic.get` return the diagnosticwith original lsp format . we can use this in code_action context .